p2p/pex/addrbook.go

```diff
@@ -938,10 +941,7 @@ func groupKeyFor(na *p2p.NetAddress, routabilityStrict bool) string {
 }
 
 func (a *addrBook) hash(b []byte) ([]byte, error) {
```

Comment:
Is there concurrent access to this?

Reply:
Yes, but it's guarded with a mutex by the caller.